features/claims/advocate/scheme_eleven/advocate_trial_claim_offences.feature

```diff
@@ -20,3 +20,15 @@ Feature: Advocate creates a claim for a final fee trial case under scheme 11
 
     And I search for a post agfs reform offence 'Harbouring escaped prisoner'
     Then I should see 'Band: 8.1'
+    And I search for a post agfs reform offence 'Burglary'
+    Then the offence should have moved from 'Band: 17.1' to 'Band: 11.2'
```

Comment:
Not a blocker but strikes me these are just a transitory test. Once fee scheme 11 is settled we will not really need these, or if it should be a an offence seeding unit test against a source of truth.